MacrobenchmarkSample/macrobenchmark/build.gradle

```
@@ -23,6 +25,17 @@ android {
     kotlinOptions {
         jvmTarget = '11'
     }
+    testOptions {
+        managedDevices {
+            devices {
+                pixel5Api30(ManagedVirtualDevice) {
+                    device = "Pixel 5"
+                    apiLevel = 30
+                    systemImageSource = "aosp"
+                }
```

NIT, let's use latest possible versions

```suggestion
                pixel6Api31(ManagedVirtualDevice) {
                    device = "Pixel 6"
                    apiLevel = 31
                    systemImageSource = "aosp"
                }
```

---

IDK too much, but okay.